bolt/test/X86/fdata-escape-chars.c

```diff
@@ -0,0 +1,69 @@
+volatile int var;
+
+void symb_w_whitespace() { var = 1; }
+
+void symb_backslash_b() {
+  symb_w_whitespace();
+  var = 2;
+}
+
+static void static_symb_backslash_b() {
+  symb_w_whitespace();
+  var = 3;
+}
+
+int main() {
+  static_symb_backslash_b();
+  symb_w_whitespace();
+  for (int i = 0; i < 2; ++i)
+    symb_backslash_b();
+}
+
+// REQUIRES: system-linux
+
+// RUN: %host_cc %cflags -O0 %s -o %t.exe -Wl,-q
+// RUN: llvm-objcopy --redefine-syms=%p/Inputs/fdata-escape-chars-syms.txt %t.exe
+//
+// RUN: llvm-bolt %t.exe -o %t.exe.instrumented -instrument \
+// RUN:   -instrumentation-file=%t.fdata
+// RUN: %t.exe.instrumented
+// RUN: cat %t.fdata | \
+// RUN    FileCheck --check-prefix="FDATA_CHECK" %s
```

Comment:
This is not being interpreted correctly by LIT. This is the output of LIT here:

  'RUN: at line 30';   cat bolt/test/X86/Output/fdata-escape-chars.c.tmp.fdata |  build/bin/llvm-bolt build/tools/bolt/test/X86/Output/fdata-escape-chars.c.tmp.exe -o tools/bolt/test/X86/Output/fdata-escape-chars.c.tmp.fdata.exe -data build/tools/bolt/test/X86/Output/fdata-escape-chars.c.tmp.fdata -print-finalized (...)
  
 It looks like it is pipeing fdata to the command in line 32? Can you check if this happens for you too?

Reply:
Yes, you are right, thank you! It was made with the help of clang-format :)